waspc/stack-stan.yaml

```diff
@@ -1,6 +1,7 @@
 resolver: ./stack-snapshot.yaml
 local-bin-path: .bin
 extra-deps:
+- stan-0.0.1.0
```

Comment:
I think this ensures that `stan` doesn't move without us deciding so, which could be tricky because extra-deps would break + we don't have reproducibility.